arangod/Cluster/ClusterInfo.cpp

```diff
@@ -296,6 +299,105 @@ class CollectionWatcher
   std::atomic<bool> _present;
 };
 
+constexpr frozen::unordered_map<std::string_view, ServerHealth, 3>
```

Comment:
Aren't there 4 entries in this map?
```suggestion
constexpr frozen::unordered_map<std::string_view, ServerHealth, 4>
```

Reply:
Strange, It should not compile in this case. Anyway, thx. Fixed.